youtube_dl/downloader/external.py

```diff
@@ -47,12 +47,20 @@ def supports(cls, info_dict):
 
     def _option(self, command_option, param):
         param = self.params.get(param)
-        if param is None:
+        if param is None or param is False:
```

Comment:
if i remove it the same problem will happen for curl and wget:
always the --insecure will be passed to curl and and --no-check-certificate to wget.
for example(i didn't pass the --no-check-certificate to youtube-dl but the insecure is passed to curl because the value of nocheckcertificate is False witch make the if `isinstance(param, bool`): pass):
```
youtube-dl -v --ignore-config --external-downloader curl https://www.youtube.com/watch?v=H8E3hLAXHRs
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-v', '--ignore-config', '--external-downloader', 'curl', 'https://www.youtube.com/watch?v=H8E3hLAXHRs']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.08.28
[debug] Python version 3.4.3 - Linux-4.1.6-1-ARCH-x86_64-with-arch-Arch-Linux
[debug] exe versions: ffmpeg 2.7.2, ffprobe 2.7.2, rtmpdump 2.4
[debug] Proxy map: {}
[youtube] H8E3hLAXHRs: Downloading webpage
[youtube] H8E3hLAXHRs: Downloading video info webpage
[youtube] H8E3hLAXHRs: Extracting video information
[youtube] H8E3hLAXHRs: Downloading DASH manifest
[youtube] H8E3hLAXHRs: Downloading DASH manifest
[debug] Invoking downloader on 'https://r6---sn-nx5cvox-2hvl.googlevideo.com/videoplayback?id=1fc13784b0171d1b&itag=133&source=youtube&requiressl=yes&pl=18&mm=31&mn=sn-nx5cvox-2hvl&ms=au&mv=m&ratebypass=yes&mime=video/mp4&gir=yes&clen=1479105&lmt=1441024129364364&dur=48.444&signature=5E5B9FDAD629A621E614855BBE23A61DAE2AB7A5.3559D2A13BCD9209576CC781830180120CB0BA53&mt=1441394976&sver=3&key=dg_yt0&fexp=9407992,9408710,9409069,9413122,9415365,9415485,9416023,9416126,9416729,9417280,9417707,9418153,9418448,9418465,9419445,9420022,9420152,9420348,9420434&upn=INxCwen4Ys4&ip=197.205.100.112&ipbits=0&expire=1441416621&sparams=ip,ipbits,expire,id,itag,source,requiressl,pl,mm,mn,ms,mv,ratebypass,mime,gir,clen,lmt,dur'
[download] Destination: خلط بين نشرة الأخبار ونشرة الأحوال الجوية على المباشر؟!-H8E3hLAXHRs.f133.mp4
False
[debug] curl command line: curl --location -o 'خلط بين نشرة الأخبار ونشرة الأحوال الجوية على المباشر؟!-H8E3hLAXHRs.f133.mp4.part' --header 'User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:10.0) Gecko/20150101 Firefox/20.0 (Chrome)' --header 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8' --header 'Accept-Charset: ISO-8859-1,utf-8;q=0.7,*;q=0.7' --header 'Accept-Language: en-us,en;q=0.5' --header 'Accept-Encoding: gzip, deflate' --insecure -- 'https://r6---sn-nx5cvox-2hvl.googlevideo.com/videoplayback?id=1fc13784b0171d1b&itag=133&source=youtube&requiressl=yes&pl=18&mm=31&mn=sn-nx5cvox-2hvl&ms=au&mv=m&ratebypass=yes&mime=video/mp4&gir=yes&clen=1479105&lmt=1441024129364364&dur=48.444&signature=5E5B9FDAD629A621E614855BBE23A61DAE2AB7A5.3559D2A13BCD9209576CC781830180120CB0BA53&mt=1441394976&sver=3&key=dg_yt0&fexp=9407992,9408710,9409069,9413122,9415365,9415485,9416023,9416126,9416729,9417280,9417707,9418153,9418448,9418465,9419445,9420022,9420152,9420348,9420434&upn=INxCwen4Ys4&ip=197.205.100.112&ipbits=0&expire=1441416621&sparams=ip,ipbits,expire,id,itag,source,requiressl,pl,mm,mn,ms,mv,ratebypass,mime,gir,clen,lmt,dur'
```